Initialise local hermit packages

juliaogris opened the pull request:
Copy upstream GitHub workflows, hermit setup and add initial hermit
package.

juliaogris committed:
Initialise local hermit-packages

Initialise local hermit-packages by copying upstream GitHub workflows
and hermit setup.

Link: https://github.com/cashapp/hermit-packages
Signed-off-by: Julia Ogris <julia.ogris@gmail.com>

camh- commented on .github/workflows/autoversion.yml:
```diff
@@ -0,0 +1,26 @@
+            git config --global user.email "github-actions[bot]@users.noreply.github.com"
+            git config --global user.name "github-actions[bot]"
+            git commit -am   "Auto-versioned: $(git diff --name-only | paste -s -d ' ' -)"
+            git push
```
will this work if the default branch (master) is protected? if so, i wonder how?

camh- reviewed (approved):
LGTM. I know my comment refers to code you copied from upstream - I wonder if it has the same branch protection settings?

Also, again, I know this was copied from upstream, but I really dislike all that shell code inside the yaml. Some of it I didn't even bother reviewing or understanding - all that stuff on one line. Some parts are redundant too (`set -e` for what is a single-line script). I would prefer it all go in a script in `bin/` (perhaps `bin/ci` and have multiple actions dispatched by the script). But that's just me.

juliaogris committed:
Add pb and servedir manifest

Add pb manifest for foxygoat/protog, we use pb for working with protog.
Add servedir manifest for foxygoat/servedir, a simple http server
serving a directories contents.

Signed-off-by: Julia Ogris <julia.ogris@gmail.com>
